driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```
@@ -1048,6 +1047,34 @@ public CCMBridge build() {
         cassandraVersion = this.version;
       }
 
+      selectedPorts.clear();
+      // Preemptively banning every number that looks like explicitly provided port
+      for (Map.Entry<String, Object> entry : this.cassandraConfiguration.entrySet()) {
+        Object value = entry.getValue();
+        try {
+          Integer result = Integer.parseInt(value.toString());
+          if ((1 <= result) && (result <= 65535)) {
```

Wouldn't it be better to throw an exception when the port is out of range?